log/log.go

```diff
@@ -15,35 +15,54 @@
 package log
 
 import (
-	"fmt"
-
-	"go.uber.org/zap"
-	"go.uber.org/zap/zapcore"
+	"github.com/bombsimon/logrusr/v2"
+	"github.com/go-logr/logr"
+	"github.com/sirupsen/logrus"
 )
 
+// TODO(log): Remove zap logic
+
 // Logger exposes logging capabilities.
 // The struct name was chosen to closely mimic other logging facilities within
 // to project to make them easier to search/replace.
 // Initial implementation was designed to encapsulate calls to `zap`, but
 // future iterations will seek to directly expose logging methods.
 type Logger struct {
-	Zap *zap.Logger
+	*logr.Logger
+	// Zap *zap.Logger
 }
 
 // NewLogger creates an instance of *zap.Logger.
 // Copied from clients/githubrepo/client.go.
 func NewLogger(logLevel Level) (*Logger, error) {
```

Comment:
Probably you are still working on it. But part of refactoring this shouldn't return `error`

Reply:
Yep, yep, I'll drop that in the next push to this PR.
If only all of our `err`s could always be `nil`! 😜